vt-digital-libraries-platform/iiif_s3_tools
# Adds script to convert flat directory of 2d, still images to a nested structure

**Adds script to convert flat directory of 2d, still images to the nested structure expected by our tiling script.**
* * *

**asana Ticket**: (https://app.asana.com/0/1203932369414348/1207150833865553) (:star:)

# What does this Pull Request do? (:star:)
Adds script to convert flat directory of 2d, still images to the nested structure expected by our tiling script.

# How should this be tested?
* run this branch ,`s3_format` on the 2d entomology collection and check if it works as expected. I'll send a copy of the updated shell script on slack.
* check for any glaring errors or oversights in the python script

# Additional Notes:
branch: `s3_format`

# Interested parties
Tag (@ mention) interested parties
@goynejennifer 

(:star:) Required fields
